### DISCUSSION TITLE
Fixed datagrid header

### DISCUSSION THREAD

**Discussion by salucien** [Megabit/Blazorise#2385] [Q&A] [answered]
Hello !
Is there a way to fix the header of a datagrid and make it always visible while the content of this datagrid is scrollable ?

**Comment by stsrki** [comment, marked as answer]
It will be possible in v0.9.4. We recently managed to add the fixed header to the Table component(PR #2293) and soon it will be done on DataGrid.

**Reply by salucien** [reply]
Ok great ! thanks :)
